runtime/runtime-params-estimator/estimate.sh

```diff
@@ -25,8 +25,8 @@ script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd "${script_dir}"/../../
 cargo run --release --package neard --bin neard -- --home /tmp/data init --chain-id= --test-seed=alice.near --account-id=test.near --fast
 cd "${script_dir}"/../../genesis-tools/genesis-populate
-cargo run --release --package genesis-populate --bin genesis-populate -- --additional-accounts-num=200000 --home /tmp/data
+cargo run --release --package genesis-populate --bin genesis-populate -- --additional-accounts-num=10000000 --home /tmp/data
```

Comment:
How this increase affects execution time of estimator under simulator?

Reply:
It does increase the time to run and increases the cost by 1.5X at least for initial lookup. But it doesn't block the estimation which is fine.